### PR TITLE
test: jarvis fiat tokens liquidity and refactors tests

### PR DESCRIPTION
**Pull request by 0xged**
I know this branch is **not** only a liquidity test, but after we talked @nchamo, I went on refactoring the liquidities test and while doing that I tried to hardcore the LEAST amount of things.
A lot of addresses in these tests were hardcoded (specially when trying to get dca-v2-core contracts) so I decided to import `chainlink-registry` and `dca-v2-core` deployments. That lead to a bit more of a refactoring.

### CODE REVIEW

File: hardhat.config.ts [hunk 1]
```diff
@@ -117,6 +117,24 @@ const config: HardhatUserConfig = {
...
+    deployments: {
+      mainnet: ['node_modules/@mean-finance/dca-v2-core/deployments/mainnet', 'node_modules/@mean-chainlink-registry-core/deployments/mainnet'],
+      mumbai: ['node_modules/@mean-finance/dca-v2-core/deployments/mumbai', 'node_modules/@mean-chainlink-registry-core/deployments/mumbai'],
+      optimism: [
+        'node_modules/@mean-finance/dca-v2-core/deployments/optimism',
+        'node_modules/@mean-chainlink-registry-core/deployments/optimism',
+      ],
+      'optimism-kovan': [
+        'node_modules/@mean-finance/dca-v2-core/deployments/optimism-kovan',
+        'node_modules/@mean-finance/chainlink-registry/deployments/optimismkovan',
```
Comment (0xged) [inline hardhat.config.ts:130]: This `optimismkovan` hurts my soul.

File: test/integration/protocols/liquidities.spec.ts [hunk 2]
```diff
@@ -0,0 +1,289 @@
...
+      });
+      then('swap is executed', async () => {
+        expect(await performedSwaps({ tokenAddress, wethAddress: WETH_ADDRESS })).to.equal(initialPerformedSwaps + 1);
+      });
+      // then('hub balance is correct', async () => {
+      //   const hubWETHBalance = await WETH.balanceOf(DCAHub.address);
+      //   const hubTokenBalance = await token.balanceOf(DCAHub.address);
+      //   expect(hubWETHBalance, 'Hub WETH balance is incorrect').to.equal(initialHubWETHBalance.sub(reward));
+      //   expect(hubTokenBalance, `Hub ${ticker} balance is incorrect`).to.equal(toProvide);
+      // });
```
Comment (0xged) [inline test/integration/protocols/liquidities.spec.ts:221]: There is something going on with the MAI test here. We can check it out together later blz?

File: deploy/000_register_hub.ts [hunk 3]
```diff
@@ -4,29 +4,29 @@ import { DeployFunction } from 'hardhat-deploy/types';
 import { networkBeingForked } from '@test-utils/evm';
 
 const deployFunction: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
```
[the remark this answers is not in the document]
Comment (nchamo) [reply deploy/000_register_hub.ts:6]: We could remove it, and next time we deploy a new chain, we publish core before trying periphery

File: test/integration/protocols/liquidities.spec.ts (hunk 2 above)
Comment (nchamo) [reply test/integration/protocols/liquidities.spec.ts:221]: Yeah, sure. Let's add it to our To Do list, so we don't forget

**Review by nchamo — APPROVED** [review]
Amazing work!!!

**Pull request merged**